src/tlo/methods/hsi_event.py

```diff
@@ -283,7 +284,7 @@ def make_appt_footprint(self, dict_of_appts) -> Counter:
     def add_equipment(self, item_codes: Union[int, str, Iterable[int], Iterable[str]]) -> None:
         """Declare that piece(s) of equipment are used in this HSI_Event. Equipment items can be identified by their
         item_codes (int) or descriptors (str); a singular item or an iterable of items (either codes or descriptors but
-        not a mix of both) can be defined at once. Checks are done on the validity of the item_codes/item 
+        not a mix of both) can be defined at once. Checks are done on the validity of the item_codes/item
```

Comment:
```suggestion
        not a mix of both) can be defined at once. Checks are done on the validity of the item_codes/item 
```

Reply:
This is an inadvertent change, but I can't seem to make a commit to roll it back (git errors saying "cannot commit"). So, I think I'm stuck with this little change hitchhiking its way on this PR.

Reply:
This is possibly just your editor autostripping whitespace at ends of line, which is generally not a bad thing so I think fine to leave, but thank you for flagging!

Reply:
Okay thanks for confirming.